ReactiveCocoaTests/Swift/SignalLifetimeSpec.swift

```diff
@@ -210,6 +210,69 @@ class SignalLifetimeSpec: QuickSpec {
 				expect(signal).to(beNil())
 			}
 		}
+
+		describe("observe") {
+			var signal: Signal<Int, TestError>!
+			var observer: Signal<Int, TestError>.Observer!
+
+			var token: NSObject? = nil
+			weak var weakToken: NSObject?
+
+			func expectTokenNotDeallocated() {
+				expect(weakToken).toNot(beNil())
+			}
+
+			func expectTokenDeallocated() {
+				expect(weakToken).to(beNil())
+			}
+
+			beforeEach {
+				let (signalTemp, observerTemp) = Signal<Int, TestError>.pipe()
+				signal = signalTemp
+				observer = observerTemp
+
+				tokenWasDeallocated = false
```

Comment:
This is still in here too, which is causing the CI failure. I should have called this out specifically in my last comment—sorry.